.github/workflows/cicd_main.yml

```diff
@@ -6,7 +6,7 @@ on:
   push:
     branches: [ main ]
   pull_request:
-    branches: [ main ]
```

Comment:
File `cicd_develop.yml` should be invoked for all PRs, I think. I'm not sure what this change does compared to other pipelines.

Reply:
This PR is against `feature/instructor-checkout-changes` rather than `develop`, so needs explicit inclusion in a pipeline somewhere. I'm not sure if `cicd_main.yml` is the best place to do this, it was just the most convenient